.github/workflows/release-ios.yml

```
@@ -1,10 +1,11 @@
 name: Release (iOS)
 
 on:
-  push:
-    branches: [ dev ]
-    paths:
-      - './Cargo.toml'
+  workflow_run:
+    workflows: [ 'Dry-run before publish' ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)